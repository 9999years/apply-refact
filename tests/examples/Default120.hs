import qualified Control.Monad 
yes = flip mapM