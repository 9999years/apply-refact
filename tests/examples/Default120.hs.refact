[("tests/examples/Default120.hs:2:7-15: Suggestion: Use forM\nFound:\n  flip mapM\nPerhaps:\n  Control.Monad.forM\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 2, startCol = 7, endLine = 2, endCol = 16}, subts = [], orig = "Control.Monad.forM"}])]
