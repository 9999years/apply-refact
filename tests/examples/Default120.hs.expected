import qualified Control.Monad
yes = Control.Monad.forM
